olga/analytics/tests/test_models.py

```diff
@@ -0,0 +1,309 @@
+"""
+Tests for analytics models.
+"""
+
+from datetime import date, datetime
+
+from ddt import ddt, data, unpack
+from mock import patch
+
+from django.test import TestCase
+
+from olga.analytics.tests.factories import EdxInstallationFactory, InstallationStatisticsFactory
+from olga.analytics.models import (
+    EdxInstallation, InstallationStatistics, get_previous_day_start_and_end_dates
+)
+
+# pylint: disable=invalid-name, attribute-defined-outside-init
+
+
+class TestEdxInstallationMethods(TestCase):
+    """
+    Tests for EdxInstallation model's core methods, that work with database.
+    """
+
+    @staticmethod
+    def setUp():
+        """
+        Create one EdxInstallation object.
+        """
+        EdxInstallationFactory(platform_url=None)
+
+    @staticmethod
+    def fake_installation_enthusiast_level():
+        """
+        Create fake edx installation sends statistics with enthusiast level.
+
+        Actually it sends a lot of data, but we need to mock only `platform_url`.
+        """
+        edx_installation_object = EdxInstallation.objects.first()
+        edx_installation_object.platform_url = 'https://url.exists'
+        edx_installation_object.save()
```

Comment:
`Update` method is intended to [update multiple objects at once](https://docs.djangoproject.com/en/1.11/topics/db/queries/#updating-multiple-objects-at-once)/=.

If I need to do this here, I should use `filter` and it will be ugly:

```python
EdxInstallation.objects.filter(pk=1)[0].update(platform_url='https://url.exists')
```